scripts/prepdocs.py

```diff
@@ -77,6 +77,14 @@ async def setup_file_strategy(credential: AsyncTokenCredential, args: Any) -> Fi
         ".pdf": FileProcessor(pdf_parser, sentence_text_splitter),
         ".json": FileProcessor(JsonParser(), SimpleTextSplitter()),
         ".docx": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".pptx": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".xlsx": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".png": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".jpg": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".jpeg": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".tiff": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".bmp": FileProcessor(doc_int_parser, sentence_text_splitter),
+        ".heic": FileProcessor(doc_int_parser, sentence_text_splitter),
```

Comment:
I added the rest of the supported formats here too. I only tested PNG manually. I'm currently assigning them sentence_text_splitter since I'm guessing folks have image screenshots of normal text, but some people might change to simpletextsplitter if that's more appropriate.